data/ayatana-indicator-notifications.service.in

```diff
@@ -4,7 +4,7 @@ PartOf=graphical-session.target
 PartOf=ayatana-indicators.target
 
 [Service]
-ExecStart=@pkglibexecdir@/ayatana-indicator-notifications-service
+ExecStart=@CMAKE_INSTALL_FULL_LIBEXECDIR@/ayatana-indicator-notifications/ayatana-indicator-notifications-service
```

Comment:
why not use @CMAKE_PROJECT_NAME@ here, as well?

Reply:
I am considering moving the executables one level up in the future (very low priority), so this would become redundant. Leaving these hard-coded help me make the TODO more apparent. No other reason.